api/turing/api/ensembling_job_api_test.go

```diff
@@ -0,0 +1,339 @@
+						Dataset: &batchensembler.Dataset{
+							Type: batchensembler.Dataset_DatasetType(
+								batchensembler.Dataset_BQ,
+							),
+							Config: &batchensembler.Dataset_BqConfig{
+								BqConfig: &batchensembler.Dataset_BigQueryDatasetConfig{
+									Query: "select * from helloworld where customer_id = 4",
+									Options: map[string]string{
+										"viewsEnabled":           "true",
+										"materializationDataset": "dataset",
+									},
+								},
+							},
+						},
+						JoinOn: []string{"customer_id", "target_date"},
+					},
+					Predictions: map[string]*batchensembler.PredictionSource{
+						"model_a": {
+							Dataset: &batchensembler.Dataset{
+								Type: batchensembler.Dataset_DatasetType(
+									batchensembler.Dataset_BQ,
+								),
+								Config: &batchensembler.Dataset_BqConfig{
+									BqConfig: &batchensembler.Dataset_BigQueryDatasetConfig{
+										Table: "project.dataset.predictions_model_a",
+										Features: []string{
+											"customer_id",
+											"target_date",
+											"predictions",
+										},
+									},
+								},
+							},
+							Columns: []string{"predictions"},
+							JoinOn:  []string{"customer_id", "target_date"},
+						},
+						"model_b": {
+							Dataset: &batchensembler.Dataset{
+								Type: batchensembler.Dataset_DatasetType(
+									batchensembler.Dataset_BQ,
+								),
+								Config: &batchensembler.Dataset_BqConfig{
+									BqConfig: &batchensembler.Dataset_BigQueryDatasetConfig{
+										Query: "select * from helloworld where customer_id = 3",
+									},
+								},
+							},
+							Columns: []string{"predictions"},
+							JoinOn:  []string{"customer_id", "target_date"},
+						},
+					},
+					Ensembler: &batchensembler.Ensembler{
+						Result: &batchensembler.Ensembler_Result{
+							ColumnName: "prediction_score",
+							Type:       batchensembler.Ensembler_FLOAT,
+							ItemType:   batchensembler.Ensembler_FLOAT,
+						},
+					},
+					Sink: &batchensembler.Sink{
+						Type: batchensembler.Sink_BQ,
+						Columns: []string{
+							"customer_id as customerId",
+							"target_date",
+							"results",
+						},
+						SaveMode: batchensembler.Sink_OVERWRITE,
+						Config: &batchensembler.Sink_BqConfig{
+							BqConfig: &batchensembler.Sink_BigQuerySinkConfig{
+								Table:         "project.dataset.ensembling_results",
+								StagingBucket: "bucket-name",
+								Options: map[string]string{
+									"partitionField": "target_date",
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+	if genExpected {
+		value.EnsemblerConfig.EnsemblerConfig.Spec.Ensembler.Uri = "/home/spark/ensembler"
+	}
+	return value
+}
+
+func createPyFuncEnsembler(id int) models.EnsemblerLike {
+	return &models.PyFuncEnsembler{
+		GenericEnsembler: &models.GenericEnsembler{
+			Model:     models.Model{ID: 1},
+			Type:      models.EnsemblerTypePyFunc,
+			ProjectID: 1,
+		},
+		ArtifactURI: "gs://d-gods-mlp/ensembler",
```

Comment:
Can you rename `d-gods-mlp` into something generic (i.e. backet-name)? Remember that that's OSS project

Reply:
Yup renamed all gojek specific stuff out.